submit-api/src/Cardano/TxSubmit/Types.hs

```diff
@@ -86,7 +98,10 @@ data TxSubmitApiRecord route = TxSubmitApiRecord
 data CBORStream
 
 instance Accept CBORStream where
-  contentType _ = "application" // "cbor"
+  contentTypes _ = "application" // "cbor" /: ("encoding", "base16") :|
```

Comment:
It could also just autodetect the type by trying each in turn. Hex isn't valid base64 isn't valid cbor.

Reply:
I consider this option indeed, but went for the explicit way which also leads to better errors. The downside auto-detection in an alternative side is that it's often hard to tell whether the input is malformed for a particular encoding or whether it is just the wrong encoding :/